deploy/demo-cli/README.md

```diff
@@ -9,15 +9,19 @@ $ ./blox-list-environments.py --help
 == Blox Demo CLI - List Blox Environments ==
 
 usage: blox-list-environments.py [-h] [--region REGION] [--apigateway]
-                                 [--host HOST]
+                                 [--host HOST] [--environment ENVIRONMENT]
```

Comment:
Correct. I didn't think it was worth it to build a completely new CLI script, when this functionality could be supported by passing in the environment to the list command.